Scripts/Python/nxusBookMachine.py

```diff
@@ -1297,6 +1310,9 @@ def IUpdateCityLinksList(self):
             #special case: and another one for GuildPub name
             elif hasattr(ageData, 'guild'):
                 displayName = "The %s' Pub" % (ageData.guild)
+            #another special case for the new GoMe Pub
+            elif ageData.ageFilename == 'GoMePubNew':
+                displayName = PtGetLocalizedString("Nexus.Messages.GoMePubNewName")
```

Comment:
You'll probably want to put this under `Global.AgeNames.GoMePubNew` 😉

Reply:
> You'll probably want to put this under `Global.AgeNames.GoMePubNew` 😉

Ah, didn't even know that was a thing. 😊  I'll make the change.